Updated top note w/ MSFT Teams Info

SamCosby opened the pull request:
Microsoft Teams does not yet support Dynamic Membership in Office 365 Groups. If the Group is initially a Dynamic Based Group then it will fail. We have confirmed this with Engineering via the following links:

https://icm.ad.msft.net/imp/v3/incidents/details/60890510/home
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/203190

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

bwren commented:
#sign-off

SamCosby committed:
Updated top note.

Microsoft Teams does not yet support Dynamic Membership in Office 365 Groups. If the Group is initially a Dynamic Based Group then it will fail. We have confirmed this with Engineering via the following links:

https://icm.ad.msft.net/imp/v3/incidents/details/60890510/home
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/203190
